corehq/apps/campaign/models.py

```diff
@@ -1 +1,11 @@
-# Create your models here.
+from django.db import models
+
+
+class Dashboard(models.Model):
```

Comment:
I chose to name the dashboard model "Dashboard" instead of "CampaignDashboard" because when it is imported (`from corehq.apps.campaign.models import Dashboard`) it is obvious that it is the campaign dashboard. The shorter name seemed preferable.